client/pcr.go

```diff
@@ -53,6 +64,35 @@ func implementedPCRs(rw io.ReadWriter) ([]tpm2.PCRSelection, error) {
 	return sels, nil
 }
 
+// Get a list of selections corresponding to the TPM's implemented PCRs
+func implementedPCRsDirect(thetpm transport.TPM) (*tpml.PCRSelection, error) {
```

Comment:
Naming nit (here and elsewhere), instead of `thetpm` just call it `t`.

Reply:
The naming of "thetpm" was chosen as that is what is consistently used in tpmDirect. See any test file: https://github.com/google/go-tpm/tree/tpmdirect/direct/tpm2 
Personally I find it easier to distinguish, and one concern with "t" is that it is a common name used in testing.